docs/reference/commandline/index.md

```diff
@@ -182,3 +182,12 @@ read the [`dockerd`](dockerd.md) reference page.
 | [plugin push](plugin_push.md) | Push a plugin to a registry                  |
 | [plugin rm](plugin_rm.md) | Remove a plugin                                  |
 | [plugin set](plugin_set.md)  | Change settings for a plugin                  |
+
+### System commands
+
+| Command | Description                                                        |
+|:--------|:-------------------------------------------------------------------|
+| [system df](system_df.md) | Show docker filesystem usage                      |
+| [system events](events.md) | Get real time events from the server            |
+| [system info](info.md) | Display system-wide information                     |
+| [system prune](system_prune.md) | Delete unused data                         |
```

Comment:
hm, yes, there's also https://docs.docker.com/engine/reference/commandline/system_info/, but that one's missing most of the detailed information. Note that this pages appears to not be used on docs.docker.com (it redirects to https://docs.docker.com/engine/reference/commandline/docker/, which has an overview, but not grouped per "object" type)

@mstanleyjones wdyt?